spec/features/guider_views_their_activities_spec.rb

```diff
@@ -34,11 +34,13 @@ def when_they_view_their_activity
   end
 
   def then_they_see_all_activities
+    first_id = @appointments.first.id
+    second_id = @appointments.second.id
     expect(@page.activities.map(&:text)).to match_array [
```

Comment:
I wonder if this is too specific by matching exactly on the rendered text of each activity? The activity classes themselves seem to be the best place to spec the generated output. Maybe we should just verify the presence of entries here with known tags (ID or something)?